sympy/functions/special/error_functions.py

```diff
@@ -2078,7 +2077,15 @@ def _eval_aseries(self, n, args0, x, logx):
                     for k in range(int(n/2) - 1)] + [Order(1/z**n, x)]
             return (sin(z)/z)*Add(*p) - (cos(z)/z)*Add(*q)
 
-        # All other points are not handled
+        # Expansion at -oo
+        elif point is S.NegativeInfinity:
+            z = self.args[0]
+            p = [S.NegativeOne**k * factorial(2*k) / z**(2*k)
+                    for k in range(int((n - 1)/2))] + [I*pi] + [Order(1/z**n, x)]
+            q = [S.NegativeOne**k * factorial(2*k + 1) / z**(2*k + 1)
+                    for k in range(int(n/2) - 1)] +[Order(1/z**n, x)]
+            return (sin(z)/z)*Add(*p) - (cos(z)/z)*Add(*q)
+
         return super(Ci, self)._eval_aseries(n, args0, x, logx)
```

Comment:
Thanks for this, I shall go through this soon. I am not sure what is the case with complex infinites like `I*oo`, give it a try if that interests you ;)

Reply:
sure sir!